python_modules/dagster/dagster_tests/definitions_tests/freshness_checks_tests/test_last_update_freshness.py

```diff
@@ -197,7 +196,7 @@ def my_asset():
             [check],
             AssetCheckSeverity.WARN,
             False,
-            description_match="Asset is overdue. Expected an update within the last 10 minutes.",
```

Comment:
Why remove this context? Is it inaccurate?

Reply:
It's just really verbose for a test, I feel. The description is still actually this verbose, we're just not checking the whole thing.

Reply:
Ahh missed that. 👍